pinot-controller/src/test/java/org/apache/pinot/controller/helix/ControllerTest.java

```diff
@@ -407,7 +407,7 @@ public static LogicalTableConfig getDummyLogicalTableConfig(String tableName, Li
         .setBrokerTenant(brokerTenant)
         .setRefOfflineTableName(offlineTableName)
         .setRefRealtimeTableName(realtimeTableName)
-        .setQuotaConfig(new QuotaConfig(null, "999"))
+        .setQuotaConfig(new QuotaConfig(null, "99999"))
```

Comment:
Is this change on purpose ? Why ?

Reply:
Yes. The `testUpdateLogicalTableTimeBoundary` send large number queries and with quota, it started returning `toomanyrequest` errors. So increased it.